examples/jsm/loaders/TaskManager.js

```diff
@@ -0,0 +1,560 @@
+/**
+ * @author Don McCurdy / https://www.donmccurdy.com
+ * @author Kai Salmen / https://kaisalmen.de
+ */
+
+import { FileLoaderBufferAsync } from "./obj2/utils/FileLoaderBufferAsync.js";
```

Comment:
Is this import from the `obj2` folder necessary?  It looks like it just adds a custom onProgress callback to the FileLoader, which may not be applicable to all implementations? If it is necessary it should probably be moved into the TaskManager file.

Reply:
The necessity for it is gone. I created it for other prototyping activity and used it because it was there. With the introduction of `loadAsync` in `Loader` it collapsed to what it is now and I agree whatever is needed (using `FileLoader` with `setResponseType( 'arraybuffer'`) should be integrated directly in `TaskManager`.

Reply:
Done